templates/org/aac/aac-article-title-content.md

```diff
@@ -10,132 +11,148 @@ _Download a [Visio file](https://arch-center.azureedge.net/architecture.vsdx) th
 
 ### Workflow
 
-> Use a numbered list if there are corresponding numbers in the diagram. If not, use a bulleted list.
+<!--
+- Title this section "Workflow" if data isn't in the scenario.
```

Comment:
The section above is titled "Workflow", so line 15 doesn't make sense. In line 19, "dataflow" seems out of place.